src/hackney_multipart.erl

```diff
@@ -67,9 +67,16 @@ encode_form(Parts, Boundary) ->
                     {ok, Bin} = file:read_file(Path),
                     PartBin = << MpHeader/binary, Bin/binary, "\r\n"  >>,
                     {AccSize1, << AccBin/binary, PartBin/binary >>};
-                ({file, Path, Disposition, ExtraHeaders}, {
+                ({file, Path, <<Name/binary>>, ExtraHeaders}, {AccSize, AccBin}) ->
```

Comment:
i would rather match it after the `{file, Path, {Disposition, Params}, ..}` clause since it will match a tuple. Also use the is_binary guard not `<<Name/binary>>`

Reply:
fixed

Reply:
thanks!